libraries/Microsoft.Bot.Builder.AI.QnA/Dialogs/QnAMakerDialog.cs

```diff
@@ -694,55 +749,6 @@ private async Task<DialogTurnResult> CheckForMultiTurnPromptAsync(WaterfallStepC
             return await stepContext.NextAsync(stepContext.Result, cancellationToken).ConfigureAwait(false);
         }
 
-        private async Task<DialogTurnResult> DisplayQnAResultAsync(WaterfallStepContext stepContext, CancellationToken cancellationToken)
```

Comment:
nit odd the way this diff is so messed up

Reply:
(this isn't actionably right now)

Reply:
This change is only on the access modifier and movement because of the method now being protected virtual instead of private. So, would not want to make any other changes to code in this PR.